modules/core/src/main/java/org/apache/ignite/internal/processors/datastreamer/DataStreamerImpl.java

```diff
@@ -316,6 +318,13 @@ public DataStreamerImpl(
         this.cacheName = cacheName;
         this.flushQ = flushQ;
 
+        if (getBoolean(IGNITE_DATA_STREAMER_ALLOW_OVERWRITE, true)) {
+            rcvr = DataStreamerCacheUpdaters.individual();
+        }
+        else {
+            rcvr = ISOLATED_UPDATER;
+        }
```

Comment:
Here, you can rewrite the part like this:
`
        if (getBoolean(IGNITE_DATA_STREAMER_ALLOW_OVERWRITE)) {
            rcvr = ISOLATED_UPDATER;
        }
        else {
            rcvr = DataStreamerCacheUpdaters.individual();
        }
`
without duplication default value.
But it is your choice.

Reply:
This approach doesn't work because default value from SystemProperty annotation is not used as default in `getBoolean` method.